client/consensus/epochs/src/lib.rs

```diff
@@ -277,20 +508,19 @@ impl<Hash, Number, Epoch> EpochChanges<Hash, Number, Epoch> where
 
 		if parent_number == Zero::zero() {
 			// need to insert the genesis epoch.
-			let genesis_epoch = make_genesis(slot_number);
-			return Ok(Some(ViableEpoch::Genesis(UnimportedGenesisEpoch(genesis_epoch))));
+			return Ok(Some(ViableEpochDescriptor::UnimportedGenesis(slot_number)))
```

Comment:
```suggestion
			return Ok(Some(ViableEpochDescriptor::UnimportedGenesis(slot_number)));
```

Reply:
Checking https://wiki.parity.io/Substrate-Style-Guide I think we encourage not having trailing `;` for `return` statement.

> * Avoid trailing ;s where unneeded.
> ```
> if condition {
> 	return 1    // <-- no ; here
> }
> ```

So I think maybe we should keep the current.